packages/playground/src/provider/index.tsx

```diff
@@ -10,27 +9,21 @@ import { ToastViewport } from './ToastViewport'
 export function Provider({ children, ...rest }: TamaguiProviderProps) {
   const scheme = useColorScheme()
   return (
-    <BottomSheetModalProvider>
-      <TamaguiProvider
-        disableInjectCSS
-        defaultTheme={scheme === 'dark' ? 'dark' : 'light'}
-        {...rest}
+    <TamaguiProvider disableInjectCSS defaultTheme={scheme === 'dark' ? 'dark' : 'light'} {...rest}>
+      <ToastProvider
```

Comment:
I wonder if we should add a toast action

Reply:
definitely! and maybe it should take ref or component for custom toast. I will create an issue for this!